script/elevate-cpanel.PL

```diff
@@ -457,8 +506,8 @@ sub monitor_upgrade {
     if ( $status eq 'success' ) {
         Elevate::Logger::INFO_nolog("# Upgrade was successful. Showing the last lines of the update log.");
     }
-    elsif ( $status eq 'running' ) {
-        Elevate::Logger::INFO_nolog("# Monitoring existing upgrade (stage=$stage) ; $tail_msg");
+    elsif ( $status eq 'running' || $status eq 'paused' ) {
```

Comment:
paused is not the right terminology for certain.

Reply:
it s generic and we can pause between any stages